bucket/bucket.py

```diff
@@ -1,9 +1,10 @@
 # SPDX-License-Identifier: MIT
 # Copyright (c) 2023 Vypercore. All Rights Reserved
 
-class Cursor:
-    # This class takes all the axes and their values
-    # and inrements the coverage for that cross
+class Bucket:
+    '''
+    This class is used for incrementing the hit count on a given bucket
+    '''
```

Comment:
Totally optional, but I find it really useful to have an example use up here for reference in the code. I think the syntax is (note double colon):
```
Blah blah blah, for example::

    code
```